### ISSUE TITLE
Error in building jdbcpool supporting Library  

### ISSUE THREAD

**Issue by ajinkyadesai** [MastekLtd/JBEAM#7]
I am using  Java version  1.7.0_2

When I  go to .../JBEAM/supporting_libraries/jdbcpool and execute command mvn clean compile install, got error as below.

[INFO] Compiling 25 source files to D:\Ajinkya\Jbeamwsnew\JBEAM\supporting_libraries\jdbcpool\target\classes
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 2.077 s
[INFO] Finished at: 2015-02-04T14:07:45+05:30
[INFO] Final Memory: 19M/309M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.0.2:compile (default-compile) on project jdbcpool: Compilation failure: Compilation failure:
[ERROR] D:\Ajinkya\Jbeamwsnew\JBEAM\supporting_libraries\jdbcpool\src\main\java\jdbc\tuning\StatementWrapper.java:[88,7] error: StatementWrapper is not abstract and does not override abstract method isCloseOnCompletion() in Statement
[ERROR] 
[ERROR] D:\Ajinkya\Jbeamwsnew\JBEAM\supporting_libraries\jdbcpool\src\main\java\jdbc\tuning\PreparedStatementWrapper.java:[123,7] error: PreparedStatementWrapper is not abstract and does not override abstract method isCloseOnCompletion() in Statement
[ERROR] 
[ERROR] D:\Ajinkya\Jbeamwsnew\JBEAM\supporting_libraries\jdbcpool\src\main\java\jdbc\tuning\CallableStatementWrapper.java:[116,7] error: CallableStatementWrapper is not abstract and does not override abstract method <T>getObject(String,Class<T>) in CallableStatement
[ERROR] 
[ERROR] could not parse error message:   where T is a type-variable:
[ERROR] T extends Object declared in method <T>getObject(String,Class<T>)
[ERROR] D:\Ajinkya\Jbeamwsnew\JBEAM\supporting_libraries\jdbcpool\src\main\java\jdbc\tuning\ConnectionWrapper.java:177: error: ConnectionWrapper is not abstract and does not override abstract method getNetworkTimeout() in Connection
[ERROR] public class ConnectionWrapper implements Connection, OracleConnection
[ERROR] ^
[ERROR] 
[ERROR] D:\Ajinkya\Jbeamwsnew\JBEAM\supporting_libraries\jdbcpool\src\main\java\jdbc\tuning\ResultSetWrapper.java:[61,7] error: ResultSetWrapper is not abstract and does not override abstract method <T>getObject(String,Class<T>) in ResultSet
[ERROR] 
[ERROR] could not parse error message:   where T is a type-variable:
[ERROR] T extends Object declared in method <T>getObject(String,Class<T>)
[ERROR] D:\Ajinkya\Jbeamwsnew\JBEAM\supporting_libraries\jdbcpool\src\main\java\jdbc\pool\JDBCDataSource.java:46: error: JDBCDataSource is not abstract and does not override abstract method getParentLogger() in CommonDataSource
[ERROR] public class JDBCDataSource implements DataSource {
[ERROR] ^
[ERROR] 
[ERROR] D:\Ajinkya\Jbeamwsnew\JBEAM\supporting_libraries\jdbcpool\src\main\java\jdbc\tuning\DatabaseMetaDataWrapper.java:[39,7] error: DatabaseMetaDataWrapper is not abstract and does not override abstract method generatedKeyAlwaysReturned() in DatabaseMetaData
[ERROR] -> [Help 1]

**Comment by shantanu-mastek** [comment]
We have tested it with java version 1.6.24 nad maven version 3.0.4. Can you check by first changing java vesrion to 1.6.*.

**Comment by ajinkyadesai** [comment]
Yes I have tested on java version 1.6 and logged the issues 8 and 9
